### PR TITLE
MTDSA-5767: Align insurance policies documentation response and request to updated specs.

### PR DESCRIPTION
**Pull request by BetaDraconis**

### CODE REVIEW

File: resources/public/api/conf/1.0/examples/insurancePolicies/retrieve_insurance_response.json [hunk 1]
```diff
@@ -1,70 +1,70 @@
...
-      "customerReference":"INPOLY123D",
-      "event":"Policy matured",
-      "gainAmount":1000.23,
-      "taxPaid":100.23,
-      "yearsHeld":10,
-      "yearsHeldSinceLastGain":1
+      "customerReference": "INPOLY123D",
+      "event": "Policy matured",
+      "gainAmount": 1000.23,
+      "taxAmountPaid": 100.23,
```
Comment (narendravijayarao) [inline resources/public/api/conf/1.0/examples/insurancePolicies/retrieve_insurance_response.json:40]: this should be ** taxPaidAmount**

File: resources/public/api/conf/1.0/examples/insurancePolicies/retrieve_insurance_response.json [hunk 2]
```diff
@@ -1,70 +1,70 @@
...
-  "foreign":[
+  "foreign": [
     {
-      "customerReference":"INPOLY123E",
-      "gainAmount":1000.23,
-      "taxPaid":100.23,
-      "yearsHeld":10
+      "customerReference": "INPOLY123E",
+      "gainAmount": 1000.23,
+      "taxAmountPaid": 100.23,
```
Comment (narendravijayarao) [inline resources/public/api/conf/1.0/examples/insurancePolicies/retrieve_insurance_response.json:49]: ** taxPaidAmount**

File: resources/public/api/conf/1.0/schemas/insurance_definitions.json [hunk 3]
```diff
@@ -28,6 +28,11 @@
       "maximum:": 99999999999.99
     },
     "taxPaid": {
+      "type": "boolean",
+      "description": "A boolean indicating whether tax has been paid.",
+      "example": "true"
+    },
+    "taxAmountPaid": {
```
Comment (narendravijayarao) [inline resources/public/api/conf/1.0/schemas/insurance_definitions.json:35]: ** taxPaidAmount**

File: resources/public/api/conf/1.0/schemas/insurance_definitions.json [hunk 4]
```diff
@@ -82,8 +88,8 @@
       "properties": {
         "customerReference": {"$ref": "#/definitions/customerReference"},
         "event": {"$ref": "#/definitions/event"},
+        "taxAmountPaid": {"$ref": "#/definitions/taxAmountPaid"},
```
Comment (narendravijayarao) [inline resources/public/api/conf/1.0/schemas/insurance_definitions.json:91]: ** taxPaidAmount**

File: resources/public/api/conf/1.0/schemas/insurance_definitions.json [hunk 5]
```diff
@@ -97,7 +103,7 @@
       "properties": {
         "customerReference": {"$ref": "#/definitions/customerReference"},
         "gainAmount": {"$ref": "#/definitions/gainAmount"},
-        "taxPaid": {"$ref": "#/definitions/taxPaid"},
+        "taxAmountPaid": {"$ref": "#/definitions/taxAmountPaid"},
```
Comment (narendravijayarao) [inline resources/public/api/conf/1.0/schemas/insurance_definitions.json:106]: ** taxPaidAmount**

File: resources/public/api/conf/1.0/examples/insurancePolicies/amend_insurance_request.json [hunk 6]
```diff
@@ -1,53 +1,53 @@
...
-  "foreign":[
+  "foreign": [
     {
-      "customerReference":"INPOLY123E",
-      "gainAmount":1000.23,
-      "taxPaid":100.23,
-      "yearsHeld":10
+      "customerReference": "INPOLY123E",
+      "gainAmount": 1000.23,
+      "taxAmountPaid": 100.23,
```
Comment (narendravijayarao) [inline resources/public/api/conf/1.0/examples/insurancePolicies/amend_insurance_request.json:49]: ** taxPaidAmount**

Comment (BetaDraconis) [reply resources/public/api/conf/1.0/examples/insurancePolicies/amend_insurance_request.json:49]: The JIRA story says taxAmountPaid

**Pull request merged**